development/tools/cmd/rendertemplates/README.md

```diff
@@ -2,18 +2,18 @@
 
 ## Overview
 
-The Render Templates is a tool that reads the configuration in the [`templates/config.yaml`](../../../../templates/config.yaml) file to generate output files, such as Prow component jobs.
+The Render Templates is a tool that reads the configuration from the [`templates/config.yaml`](../../../../templates/config.yaml) file and data files from [`templates/data`](../../../../templates/data) directory to generate output files, such as Prow component jobs. While `config.yaml` file can hold configuration for output file, you can place such data within data files, which should hold configuration for related output files. Having separate files with grouped data is cleaner and easier to maintain than one huge config file.
 
-The `config.yaml` file specifies the following for the Render Templates:
+The `config.yaml` file and `data files` specifies the following for the Render Templates:
 - Templates it must use to generate the output files
 - The name and location of the output files
 - Values it must use to generate the output files
```

Comment:
Since you've added one more thing to the preceding line, we now refer to a plural, so `it` -> `they`/`them`. Unless it refers to the Render Templates, then it should have been plural from the start. 
```suggestion
- Templates they must use to generate the output files
- The name and location of the output files
- Values they must use to generate the output files
```

Reply:
Render Templates is a tool, that's why I used "it". It (Render Templates) must use templates to generate the output files.

Reply:
I wasn't sure if you wanted to refer to it as a tool or as templates, as you used both singular and plural verbs throughout the document. That convinces me. Unfortunately, that means that we need to revert some of my corrections 
![image](https://user-images.githubusercontent.com/53601578/107653334-cb296200-6c81-11eb-9783-339cb65d925f.png)